stomp.go

```diff
@@ -14,7 +14,7 @@ func StartStomp(config *Config, queue *ConfirmationQueue) {
 	stompUser := config.StompUser
 	stompPassword := config.StompPassword
 	stompUrl := config.StompURL
-	stompTopic := config.StompTopic
+	stompTopic := "/topic/" + config.StompTopic
```

Comment:
Maybe, for sanity, you could check of the `config.StompTopic` starts with `/topic/`, and prepend it if it does not.